### DISCUSSION TITLE
How to list all blocks of a page with page id?

### DISCUSSION THREAD

**Discussion by kaedea** [ramnes/notion-sdk-py#71] [Q&A] [answered]
Is that possible we list all blocks and subpages/links of a given page, and then turn them into MD files (I have finished the converting job and what I need is the page blocks data)?

**Comment by ramnes** [comment, marked as answer]
Yes: `client.blocks.children.list(block_id=page_id)`

**Reply by bergercookie** [reply]
It feels a bit counterintuitive as a path and also the fact that you have to pass a page_id as a block_id. I would ideally expect there to be a method under `pages` for this :/

**Reply by ramnes** [reply]
Feel free to tell that to Notion employees. :)

Email: developers@makenotion.com
Slack: https://join.slack.com/t/notiondevs/shared_invite/zt-vkinpzs0-hJSEAAxJzPMuiCNvZETF2A
